### PR TITLE
Elasticsearch: Rename public/app/plugins/datasource/elasticsearch/elastic_response.ts to follow convention

### LINKED ISSUE #55384: Elasticsearch: Rename public/app/plugins/datasource/elasticsearch/elastic_response.ts to follow convention
**Level: easy**

Some files that are maintained by logs squad have snake_case name. According to convention https://github.com/grafana/grafana/blob/765b995b1bcafe8b94d0bbe53737ea6c751d63c9/contribute/style-guides/frontend.md#file-and-directory-naming-conventions we should use:
- `camelCase` when the primary export is a function
- `PascalCase` when the primary export is a class or React component. 

Rename following files  in `grafana/grafana` to follow the above-mentioned convention:
- `public/app/plugins/datasource/elasticsearch/elastic_response.ts`
- `public/app/plugins/datasource/elasticsearch/specs/elastic_response.test.ts` (this should be also moved outside of specs folder)

### PR DESCRIPTION
**Pull request by mvsousa**
**What this PR does / why we need it**:

Rename public/app/plugins/datasource/elasticsearch/elastic_response.ts to follow convention

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana/issues/55384

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ngalert/eval/evaluator_mock.go [hunk 1]
```diff
@@ -51,10 +51,10 @@ type FakeEvaluator_ConditionEval_Call struct {
 }
```
Comment (matyax) [inline pkg/services/ngalert/eval/evaluator_mock.go:51]: This change is out of scope with the issue.

**Review by matyax — CHANGES_REQUESTED** [review]
There's a modified go file.

**Review by matyax — DISMISSED** [review]
Thank you for updating the PR! The CI is failing with the following error:

```
+ yarn run prettier:check

+ yarn run lint


/drone/src/public/app/plugins/datasource/elasticsearch/ElasticResponse.test.ts

  4:1  error  There should be no empty line within import group           import/order

  4:1  error  `./types` import should occur after import of `./queryDef`  import/order


✖ 2 problems (2 errors, 0 warnings)

  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by mvsousa** [comment]
> yarn run lint

@matyax 
I think I fixed, can you check ?
I ended up messing up with the main branch and requested review from a lot of people

**Comment by matyax** [comment]
Some more conflicts to resolve.

**Comment by mvsousa** [comment]
> Some more conflicts to resolve.

@matyax done! updated with the main branch!

**Review by ivanahuckova — APPROVED** [review]
Nice! Thank you for your contribution. LGTM! 🚀

**Pull request merged**